Seqs_processing_tool/Seqs_processing_tool.py

```diff
@@ -0,0 +1,85 @@
+from bioutils.module_1_dna_rna_tools import (
+    transcribe,
+    reverse,
+    complement,
+    reverse_complement,
+    gc_counter,
+    nucacid_type,
+    is_aug_in_rna,
+)
+
+from bioutils.module_2_fastq import is_gc_fit, is_length_fit, is_quality_fit
+from bioutils.module_3_input_output import fastq_transfom_dict, dict_transform_fastq
+
+
+def run_dna_rna_tools(*args: str) -> str | list:
+    """
+    Description of function run_dna_rna_tools
+
+    :param args: list of DNA/RNA sequences. Last element of this list is action
+    :return: processed seqs made by action (if one seq -> reurn str,
+    else -> return list of str)
```

Comment:
Not really clear, which type of data to expect in case of multiple seqs, might be important for downstream procedures.